controls/ccn_rhel9.yml

```diff
@@ -0,0 +1,824 @@
+    related_rules:
+      - no_password_auth_for_systemaccounts
+      - no_shelllogin_for_systemaccounts
+
+  - id: A.6.SEC-RHEL4
+    title: The use of sessions with the "root" user is restricted
+    original_title: Se restringe el uso de sesiones con usuario "root".
+    levels:
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - ensure_root_password_configured
+      - no_empty_passwords_etc_shadow
+
+  - id: A.6.SEC-RHEL5
+    title: The global system mask is modified to be more restrictive
+    original_title: Se modifica la máscara global del sistema para ser más restrictiva.
+    levels:
+      - advanced
+    status: planned
+    related_rules:
+      - accounts_umask_etc_bashrc
+      - accounts_umask_etc_login_defs
+      - accounts_umask_etc_profile
+      - var_accounts_user_umask=027
+
+  - id: A.6.SEC-RHEL6
+    title: Unnecessary groups and users are removed from the system
+    original_title: Se eliminan los grupos y usuarios innecesarios del sistema.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: manual
+
+  - id: A.8.SEC-RHEL1
+    title: Control who can install software on the system
+    original_title: Se controla quién puede instalar software en el sistema.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: pending
+
+  - id: A.8.SEC-RHEL2
+    title: The operating system is updated
+    original_title: El sistema operativo está actualizado.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: manual
+    related_rules:
+      - security_patches_up_to_date
+
+  - id: A.8.SEC-RHEL3
+    title: The system has an activated local firewall
+    original_title: El sistema tiene un firewall local activado.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - service_firewalld_enabled
+      - service_nftables_disabled
+      - set_firewalld_default_zone
+      - firewalld_loopback_traffic_trusted
+      - firewalld_loopback_traffic_restricted
+
+  - id: A.8.SEC-RHEL4
+    title: Unnecessary services are disabled, reducing the attack surface
+    original_title: Se deshabilitan servicios innecesarios, reduciendo la superficie de exposición.
+    levels:
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - package_bind_removed
+      - package_cyrus-imapd_removed
+      - package_dovecot_removed
+      - package_net-snmp_removed
+      - package_squid_removed
+      - package_telnet-server_removed
+      - package_tftp-server_removed
+      - package_vsftpd_removed
+      - kernel_module_squashfs_disable
+      - kernel_module_udf_disabled
+
+  - id: A.8.SEC-RHEL5
+    title: Application execution is controlled
+    original_title: Se controla la ejecución de aplicaciones.
+    levels:
+      - advanced
+    status: pending
+    notes: |-
+      This might be related to SELinux.
```

Comment:
Or fapolicyd for application control.

Reply:
Good. We have to clarify these `pending` requirements at some point.

Reply:
Included